packages/x-priority-queue/src/x-priority-queue.ts

```diff
@@ -91,7 +91,7 @@ export class XPriorityQueue<Key = string> {
    *
    * @internal
    */
-  protected comparatorFn: AnyFunction<boolean>;
+  protected comparatorFn: AnyFunction;
```

Comment:
I guess changes in this file are not intended

Reply:
we have problem with types in this file, if you change that and run the test, you will see that an error will be launched.